stable/kubecost/Chart.yaml

```diff
@@ -3,6 +3,6 @@ appVersion: 1.79.1
 description: Kubecost
 name: kubecost
 home: https://github.com/mesosphere/charts
-version: 0.10.0
+version: 0.11.0
```

Comment:
@gracedo was this version bump wrong? it looks like the latest is to 0.12.0?

Reply:
it was bumped before another pr was opened which bumped to 0.11.0, so I think mister bot actually did everything properly here